src/main/webapp/styles/style.css

```diff
@@ -1,5 +1,9 @@
 .hidden {
-    display: none;
+    display: none !important;
```

Comment:
is there no way to avoid the !important tag here? It is usually not considered good practice.

Reply:
Yeah I didn't have it before but I added it because my div had a display: flex attribute and I needed to override that with display: none.